### ISSUE TITLE
Error on https://www.minijuegos.com/helpers/game/sandbox.php

### ISSUE THREAD

**Issue by LuisUrbe** [ruffle-rs/ruffle#10843]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:602:51
Error stack:
```
Error: panicked at 'called `Option::unwrap()` on a `None` value', core/src/bitmap/bitmap_data.rs:602:51
    at O.n.wbg.__wbg_new_15d3966e9981a196 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.b23fa269e4f1e918d88d.js:1:48745)
    at core::ops::function::FnOnce::call_once{{vtable.shim}}::hd28f65a61f685e47 (wasm://wasm/02b61d02:wasm-function[8383]:0x80c16a)
    at std::panicking::rust_panic_with_hook::h7f7102b82d51338f (wasm://wasm/02b61d02:wasm-function[5498]:0x72f1b8)
    at core::panicking::panic_fmt::he4489d678d6570d5 (wasm://wasm/02b61d02:wasm-function[8238]:0x8069e8)
    at core::panicking::panic::hea78c4e8b318407d (wasm://wasm/02b61d02:wasm-function[7542]:0x7cc5f7)
    at ruffle_core::bitmap::bitmap_data::BitmapData::update_dirty_texture::h69e242bde85db84e (wasm://wasm/02b61d02:wasm-function[1281]:0x3d5f88)
    at <ruffle_core::library::MovieLibrarySource as ruffle_render::bitmap::BitmapSource>::bitmap_handle::h218cbf3363c21bf3 (wasm://wasm/02b61d02:wasm-function[3241]:0x61df78)
    at <ruffle_render_wgpu::backend::WgpuRenderBackend<T> as ruffle_render::backend::RenderBackend>::register_shape::haa5f47cd0e691a1c (wasm://wasm/02b61d02:wasm-function[1119]:0x38389f)
    at ruffle_core::display_object::movie_clip::MovieClipData::define_shape::h3ce1a0757a2ce061 (wasm://wasm/02b61d02:wasm-function[3597]:0x66c051)
    at ruffle_core::display_object::movie_clip::MovieClip::preload::h30e5eba151b7c315 (wasm://wasm/02b61d02:wasm-function[620]:0x1e0cd4)
```

# Player Info
Allows script access: false

# Page Info
Page URL: https://www.minijuegos.com/helpers/game/sandbox.php?mp_assets=https%3A%2F%2Fs2.minijuegosgratis.com%2F&mp_embed=0&mp_game_id=21595&mp_game_uid=tu-95&mp_game_url=https%3A%2F%2Fwww.minijuegos.com%2Fembed%2Ftu-95&mp_int=1&mp_locale=es_ES&mp_player_type=FLASH&mp_player_url=aHR0cHM6Ly93d3cubWluaWp1ZWdvc2dyYXRpcy5jb20vZmxhc2gxMjM0L25vX2hvdGxpbmsvdHU5NS5zd2Y%3D&mp_site_https_url=https%3A%2F%2Fwww.minijuegos.com%2F&mp_site_name=minijuegos.com&mp_site_url=https%3A%2F%2Fwww.minijuegos.com%2F&mp_timezone=America%2FCaracas&mp_view_type=
SWF URL: https://www.minijuegosgratis.com/flash1234/no_hotlink/tu95.swf

# Browser Info
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/112.0.0.0 Safari/537.36
Platform: Win32
Has touch support: false

# Ruffle Info
Version: 0.1.0
Name: nightly 2023-04-27
Channel: nightly
Built: 2023-04-27T00:19:14.885Z
Commit: e5caf0fda82f408af6f2b4f8d05c44603b037eee
Is extension: false

# Metadata
width: 670
height: 450
frameRate: 24
numFrames: 31
swfVersion: 9
backgroundColor: #000000
isActionScript3: false
uncompressedLength: 3416455

**Comment by Lord-McSweeney** [comment]
Warning before registering bitmap: `WARN core/src/bitmap/bitmap_data.rs:520 Failed to register raw bitmap for BitmapData: TooLarge`

**Comment by Lord-McSweeney** [comment]
Fixed by #10908.
